cloudwatch_metrics_collector/src/metrics_collector_parameter_helper.cpp

```diff
@@ -161,16 +163,42 @@ void ReadStorageResolution(
 
 void ReadTopics(
         std::shared_ptr<Aws::Client::ParameterReaderInterface> parameter_reader,
-        std::vector<std::string> & topics) {
+        std::vector<TopicInfo> & topics) {
```

Comment:
It'd be preferable to split the logging logic from the rest (applying the extract method pattern).

Reply:
Done.